lib/_tls_wrap.js

```diff
@@ -1366,6 +1370,14 @@ Server.prototype.setTicketKeys = function setTicketKeys(keys) {
   this._sharedCreds.context.setTicketKeys(keys);
 };
 
+Server.prototype.getNumTickets = function getNumTickets() {
+  return this._sharedCreds.context.getNumTickets();
+};
+
+
+Server.prototype.setNumTickets = function setNumTickets(numTickets) {
+  this._sharedCreds.context.setNumTickets(numTickets);
```

Comment:
This should do argument type checking on `numTickets` before passing it down to the C++ layer.

Reply:
I'd generally prefer to see these as a getter/setter rather than functions

Reply:
This already does typechecking at the C++ layer, it’s not necessary here.

Reply:
Yeah, I saw that, my preference would be for that to be moved to the javascript side.

Reply:
If you look around this file, the typechecking is is done in the C++ layer for most of these functions. C++ can do stricter typechecking than JS. Any reason it should be done in JS other than preference?

Reply:
It's preference. So consider it non-blocking.

Reply:
Then will leave it as is.